www/js/app.js

```diff
@@ -6,6 +8,7 @@ require("angular-sanitize");
 
 angular.module('app', ['ionic', 'app.controllers', 'app.routes', 'app.services'])
 .constant('$fh', require("fh-js-sdk"))
+.constant('moment', require("moment"))
```

Comment:
I got an exception in preview with the JS error
services.js:15 Uncaught ReferenceError: moment is not defined